ui/helpers/constants/browser.js

```diff
@@ -0,0 +1,4 @@
+export const OUTDATED_BROWSER_VERSIONS = {
+  chrome: '<80',
+  firefox: '<78',
+};
```

Comment:
May be we can use existing constant files and not add new just for browser constants.